src/core/default/commands/function/logs.py

```diff
@@ -1,7 +1,8 @@
-from argparse import ArgumentParser
-import time, math, datetime
+import time, datetime
```

Comment:
Can you separate this file into a its own PR? Im going to just double check it by testing it out, so it would help to have it be the only change in a given PR. Its just a bit too much for me to eye check. Once I have some time, I'll figure out a way to hook up commands to an automated test suite so that we don't have to manually test them.